Dockerfile

```diff
@@ -1,47 +1,15 @@
-FROM python:3.8.6-alpine as base
+FROM python:3.8.16-slim
 
-####
+RUN apt update && \
+  apt install -y git cmake libmagic-dev libgtk-3-dev
 
-FROM base as builder
+WORKDIR /app
+COPY requirements.txt .
+RUN pip install -U pip setuptools
```

Comment:
Better to upgrade `wheel` instead of `setuptools`, because more and more packages are published _*.whl_ files, and _pip_ prefers to install packages from _*.whl_ files than *setup.py*.